client/analytics/components/report-summary/index.js

```diff
@@ -6,10 +6,6 @@ import { Component } from '@wordpress/element';
 import { compose } from '@wordpress/compose';
 import PropTypes from 'prop-types';
 
-/**
```

Comment:
such a tiny thing and I don't think it matters a lot but, is it possible to strip the newline before the comment block as well? 😅 No pressure if its a pain.

Reply:
Yeah I tried getting my replace all to work with a regex, but it was becoming too time consuming. Instead I fixed manually where I could. 

I still have my script, I bet I could re purpose it to find these.

Reply:
Woohoo, automation to the rescue. I remove in https://github.com/woocommerce/woocommerce-admin/pull/4972/commits/8dd7971777ff7007612d915e2b60f3ca60541709